course_content/solutions/numpy_exercise_2.ipynb

```diff
@@ -24,10 +25,9 @@
       "     \\int_{a}^{b}f(x)dx\n",
       "$$\n",
       "\n",
-      "we can partition the integration interval $[a,b]$ into smaller subintervals,\n",
-      "and approximate the area under the curve for each subinterval by the area of\n",
-      "the trapezoid created by linearly interpolating between the two function values\n",
-      "at each end of the subinterval."
+      "we can partition the integration interval $[a,b]$ into smaller subintervals. We then approximate the area under the curve for each subinterval by calculating the area of the trapezoid created by linearly interpolating between the two function values at each end of the subinterval:\n",
```

Comment:
It was different to both the main NumPy course and the rest of this notebook. And, for having adjusted the width of my browser it rendered weirdly in edit mode :stuck_out_tongue_winking_eye: But I see that you have merged anyway so I guess it's not too disastrous a change!